README.md

```diff
@@ -58,4 +58,9 @@ By default,
 
 The backend will then be hosted at [http://0.0.0.0:8000](http://0.0.0.0:8000). 
 
-Open [http://0.0.0.0:8000/docs](http://0.0.0.0:8000/docs) for the API docs.
\ No newline at end of file
+Open [http://0.0.0.0:8000/docs](http://0.0.0.0:8000/docs) for the API docs.
+
```

Comment:
Change to an introduction as follows (replace the description below for use cases):
The OpenROAD chat assistant aims to provide easy and quick access to information regarding tools, responses to questions and commonly occurring problems in OpenROAD and its native flow-OpenROAD-flow-scripts.

The current architecture is based on retrieval techniques from OpenROAD documentation and other online data sources. We aim to continuously improve the architecture and the associated the dataset to improve accuracy, coverage  and robustness.

Reply:
Done, please check now.